src/Products/PloneMeeting/browser/views.py

```diff
@@ -1316,40 +1316,56 @@ def print_signatures_by_position(self, **kwargs):
             return self.print_signatories_by_position(**kwargs)
 
     def print_signatories_by_position(self,
-                                      signature_format=(u'position_type', u'person'),
-                                      separator=u', ',
-                                      ender=''):
+                                      signature_format=(u'prefixed_position_type', u'person'),
+                                      separator=u',',
+                                      ender=u''):
         """
         Print signatories by position
+        :param signature_format: can be 'position_type', 'prefixed_position_type', 'person',
+        'person_title', 'secondary_position_type', 'prefixed_secondary_position_type',
+        an attribute of PMHeldPosition or a str (in this case it just print the str)
+        :param separator: str that will be appended at the end of each line (except the last one)
```

Comment:
Not clear what's the difference between prefixed_position_type and position_type.
Please add a bit more explanation for every possible values of signature_format parameter and make it so the differences between them are obvious

Reply:
Done :ok_hand: